vue/src/vue-elements/accounts-selector-panel.vue

```diff
@@ -141,6 +189,10 @@
 				//this.get_lang();
 			}
 		},
+		mounted: function () {
+			this.setupData();
+		//	this.refresh_language_taxonomies();
```

Comment:
Is this required or should be removed?

Reply:
I will removed based on this commit https://github.com/Codeinwp/tweet-old-post/commit/ca44ce0d883238006cc0bfa08ae296fdf532fa32